hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/schedule/HapiScheduleCreate.java

```diff
@@ -172,12 +172,12 @@ public HapiScheduleCreate<T> waitForExpiry(boolean value) {
 
     public HapiScheduleCreate<T> expiringAt(final long expiry) {
         this.longTermExpiry = expiry;
-        return waitForExpiry();
+        return this;
```

Comment:
This file seem to be different from original PR. Is this intentional ?

Reply:
Yes Neeha, this is intentional.
 - The `release/0.57` branch didn't yet have all the tests I brought in from `develop`; and,
 - Some of those tests assume this behavior from `HapiScheduleCreate`.